source/renderer/app/components/assets/Asset.tsx

```diff
@@ -209,7 +209,9 @@ class Asset extends Component<Props, State> {
               })}
               className={styles.warningIconWrapper}
             >
-              <SVGInline className={styles.warningIcon} svg={warningIcon} />
+              <span data-testid="warning-icon">
```

Comment:
We might want to start accepting these in polymorph, otherwise they don't get picked up if it's a non-native element.